ign_ros2_control/src/ign_ros2_control_plugin.cpp

```diff
@@ -291,19 +291,20 @@ void IgnitionROS2ControlPlugin::Configure(
     controllerManagerNodeName = controllerManagerPrefixNodeName + "_" + controllerManagerNodeName;
```

Comment:
Can we remove this logic here completely, since we would rather like to use namespaces. Isn't this kind-off doubled?